readme.md

```diff
@@ -660,6 +660,31 @@ As of version `5.0.0` it uses source maps to report coverage for your actual cod
 
 ## FAQ
 
+### How do I test HTTP endpoints?
+
+AVA doesn't have an official assertion library for endpoints, but a great option is [`supertest-as-promised`](https://www.npmjs.com/package/supertest-as-promised).
+Since the tests run concurrently, it's a best practice to create a fresh server instance for each test because if we referenced the same instance, it could be mutated between tests. This can be accomplished with a `beforeEach` and `context`, or even more simply with a factory function:
+```
+function makeApp() {
+  const app = express();
+  app.post('/signup', signupHandler);
+  return app;
+}
+```
+
+Next, just inject your server instance into supertest. The only gotcha is to use a promise or ES7 syntax instead of supertest's `end` method:
```

Comment:
Then I would write that instead. ES7 doesn't mean anything, it's ES2016, but async functions are not yet part of it.